packages.html

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<div data-template="templates:surround" data-template-with="templates/page-inner.html" data-template-at="container-body">
+<div data-template="templates:surround" data-template-with="templates/page.html" data-template-at="container-body">
     <div class="row">
         <div class="col-md-12">
             <h1>Public Application Repository</h1>
             <p>
-                <a href="../">Back to listing of all packages</a>
+                <a href="./">Back to listing of all packages</a>
```

Comment:
from `packages/something` the link back should still be `../` no?

Reply:
In my testing of the app, this change (from `../` to `./`) works.